src/service/calculation_service.py

```diff
@@ -29,10 +29,12 @@ def equalpress(self):
         try:
 
             # evaluate the expression/calculation
-            if "/" not in self.expression:
-                total = str(eval(self.expression))
-            else:
+            if "/" in self.expression:
                 total = str(divide_service.divide(self.expression))
+            if "+" in self.expression:
```

Comment:
Turn this into an else if to not get unexpected behaviours when the input is `+12 / 20`
```suggestion
            elseif "+" in self.expression:
```